app/src/hooks/AuthHandler.js

```diff
@@ -38,6 +39,15 @@ const AuthHandler = (onComplete) => {
         window.uid = authData?.uid;
         localStorage.setItem("__rq_uid", authData?.uid);
 
+        if (user.metadata.creationTime === user.metadata.lastSignInTime) {
```

Comment:
doesn't seem reliable, can we use `result.additionalUserInfo.isNewUser` instead?

Reply:
and have we tested if this is working as expected, maybe using console logs?

Reply:
@RuntimeTerror10 with [the updated changes](https://github.com/requestly/requestly/pull/1469/commits/af6d33ebc4c2639bbfff6ec59b83c07248725b9a) this check no longer seems required